.env.example

```
@@ -1,62 +1,6 @@
-INDEXER_LOG_LEVEL=info
-INDEXER_DATABASE_URL=postgres://postgres:postgres@localhost:5432/fundable_indexer
-SOROBAN_RPC_URL=https://soroban-testnet.stellar.org
-STELLAR_NETWORK_PASSPHRASE=Test SDF Network ; September 2015
-POLL_INTERVAL_MS=5000
-START_LEDGER=
-
-# Comma-separated contract IDs. Keep production values out of commits unless
-# they are intended to be public configuration.
-STREAMS_CONTRACT_IDS=
-DISTRIBUTIONS_CONTRACT_IDS=
+JWT_SECRET=super_secret_long_string_at_least_32_chars
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the PORT key

(UnorderedKey)